demo/chbench/mzcompose.yml

```diff
@@ -402,7 +402,7 @@ mzconduct:
       - step: start-services
         services: [dashboard]
       - step: start-services
-        services: [kafka, schema-registry]
+        services: [kafka, schema-registry, control-center]
```

Comment:
you might want to wait for control-center to come up as well? i don't think its mandatory so tioli but it might be nice and also parallels waiting for the other stuff

Reply:
Thanks, will add!

Reply:
Done!